Resources/Storage/StorageFactory.py

```diff
@@ -254,16 +282,14 @@ def _getConfigStorageProtocolDetails( self, storageName, protocolSection ):
     options = res['Value']
 
     # We must have certain values internally even if not supplied in CS
-    protocolDict = {'Access':'', 'Host':'', 'Path':'', 'Port':'', 'Protocol':'', 'PluginName':'', 'SpaceToken':'', 'WSUrl':''}
+    protocolDict = {'Access':'', 'Host':'', 'Path':'', 'Port':'', 'Protocol':'', 'SpaceToken':'', 'WSUrl':''}
     for option in options:
       configPath = cfgPath( protocolConfigPath, option )
       optionValue = gConfig.getValue( configPath, '' )
       protocolDict[option] = optionValue
 
-    # This is a temporary for backward compatibility
-    if "ProtocolName" in protocolDict and not protocolDict['PluginName']:
-      protocolDict['PluginName'] = protocolDict['ProtocolName']
-    protocolDict.pop( 'ProtocolName', None )
+    # This is a temporary for backward compatibility: move ProtocolName to PluginName
+    protocolDict.setdefault( 'PluginName', protocolDict.pop( 'ProtocolName', None ) )
```

Comment:
It's irrelevant as this is in case both are not defined and then we exit with an error anyway!